### ISSUE TITLE
PHP PEAR not working on vagrant up

### ISSUE THREAD

**Issue by Fynkymynky** [puphpet/puphpet#2280]
On my linux box running the vagrant 1.8 and latest Virtual box, I get the following errors with PEAR when I try and do vagrant up. When I provision the box again, it's fine. The error is when I try and load the box again.

```
==> woof: Configuring proxy for PEAR...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

/usr/bin/pear config-set http_proxy cache.foo.com:80 system

Stdout from the command:


Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: honorsbaseinstall in Role.php on line 173

Notice: Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: installable in Role.php on line 139

Notice: Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: phpfile in Role.php on line 204

Notice: Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Notice: Undefined index: config_vars in Role.php on line 46

Notice: Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in PEAR/Command.php on line 249

Warning: Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
XML Extension not found


Stderr from the command:

PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: honorsbaseinstall in /usr/share/php/PEAR/Installer/Role.php on line 173
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: installable in /usr/share/php/PEAR/Installer/Role.php on line 139
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: phpfile in /usr/share/php/PEAR/Installer/Role.php on line 204
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Notice:  Undefined index: config_vars in /usr/share/php/PEAR/Installer/Role.php on line 46
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249
PHP Warning:  Invalid argument supplied for foreach() in /usr/share/php/PEAR/Command.php on line 249

```

**Comment by spekary** [comment]
Ditto. pecl fails too as a result of this.

My config file:

vagrantfile:
    target: local
    vm:
        provider:
            local:
                box: puphpet/ubuntu1404-x64
                box_url: puphpet/ubuntu1404-x64
                box_version: '0'
                chosen_virtualizer: virtualbox
                virtualizers:
                    virtualbox:
                        modifyvm:
                            natdnshostresolver1: false
                        showgui: 0
                    vmware:
                        numvcpus: 1
                    parallels:
                        use_linked_clone: 0
                        check_guest_tools: 0
                        update_guest_tools: 0
                machines:
                    vflm_x253fyl0xu3f:
                        id: ryaaUb
                        hostname: ryaa.ub
                        network:
                            private_network: 192.168.56.101
                            forwarded_port:
                                vflmnfp_9jk9p0zg1ojo:
                                    host: '9338'
                                    guest: '22'
                        memory: '512'
                        cpus: '1'
        provision:
            puppet:
                manifests_path: puphpet/puppet/manifests
                module_path: puphpet/puppet/modules
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
        synced_folder:
            vflsf_imbsu1o5tz6u:
                source: /Sites/home
                target: /var/www
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
            vflsf_n93b6mwb2zae:
                source: /Sites/ryaa
                target: /home/ryaa
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
            vflsf_o0nquqbvocr9:
                source: /Sites/nopro
                target: /home/nopro
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
        usable_port_range:
            start: 10200
            stop: 10500
        post_up_message: ''
    ssh:
        host: 'false'
        port: 'false'
        private_key_path: 'false'
        username: vagrant
        guest_port: 'false'
        keep_alive: '1'
        forward_agent: 'false'
        forward_x11: 'false'
        shell: 'bash -l'
        insert_key: 'false'
    vagrant:
        host: detect
    proxy:
        http: ''
        https: ''
        ftp: ''
        no_proxy: ''
server:
    install: '1'
    packages:
        - nano
        - vim
users_groups:
    install: '1'
    groups:
        - ryaa
        - nopro
    users:
        - 'ryaa:ryaa:www-data'
        - 'nopro:nopro:www-data'
locale:
    install: '1'
    settings:
        default_locale: en_US.UTF-8
        locales:
            - en_GB.UTF-8
            - en_US.UTF-8
        timezone: America/Los_Angeles
firewall:
    install: '1'
    rules: {  }
cron:
    install: '1'
    jobs: {  }
nginx:
    install: '0'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
        proxy_connect_timeout: 600s
        proxy_send_timeout: 600s
        proxy_read_timeout: 600s
        names_hash_bucket_size: 128
    upstreams: {  }
    vhosts:
        nxv_pxquyaswu7zj:
            server_name: awesome.dev
            server_aliases:
                - www.awesome.dev
            www_root: /var/www/awesome
            listen_port: '80'
            index_files:
                - index.html
                - index.htm
                - index.php
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            ssl_protocols: ''
            ssl_ciphers: ''
            rewrite_to_https: '1'
            spdy: '1'
            locations:
                nxvl_1gslv9wt11rn:
                    location: /
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: ''
                    fastcgi_index: ''
                    fastcgi_split_path: ''
                    proxy: ''
                    proxy_redirect: ''
                nxvl_6332dqu14hh4:
                    location: '~ \.php$'
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: '127.0.0.1:9000'
                    fastcgi_index: index.php
                    fastcgi_split_path: '^(.+\.php)(/.*)$'
                    fast_cgi_params_extra:
                        - 'SCRIPT_FILENAME $request_filename'
                        - 'APP_ENV dev'
                    set:
                        - '$path_info $fastcgi_path_info'
                    proxy: ''
                    proxy_redirect: ''
    proxies: {  }
apache:
    install: '1'
    settings:
        version: 2.4
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - proxy_fcgi
        - rewrite
    vhosts:
        av_gfwcfvl2xxwt:
            servername: ryaa.dev
            docroot: /home/ryaa/public_html
            port: '80'
            setenv:
                - 'APP_ENV dev'
            setenvif:
                - 'Authorization "(.*)" HTTP_AUTHORIZATION=$1'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_2ks38o9ccbq4:
                    path: /home/ryaa/public_html
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_qwmlso9wosxj:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9006'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
        av_us6t08lgvns6:
            servername: home.dev
            docroot: /var/www
            port: '80'
            setenv:
                - 'APP_ENV dev'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_74g60v5lue2j:
                    path: /var/www
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_h1rzl3re3in5:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9006'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
        av_udkyer584m7a:
            servername: nopro.dev
            serveraliases:
                - www.nopro.dev
            docroot: /home/nopro/public_html
            port: '80'
            setenv:
                - 'APP_ENV dev'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_gqlhl6gsx2ah:
                    path: /home/nopro/public_html
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_l0579231c5yv:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9006'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
letsencrypt:
    install: '1'
    settings:
        email: ''
    domains: {  }
php:
    install: '1'
    settings:
        version: '56'
    modules:
        php:
            - cli
            - intl
            - mcrypt
            - imagick
            - curl
            - gd
            - mbstring
        pear: {  }
        pecl:
            - APCu
    ini:
        display_errors: 'On'
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        session.gc_maxlifetime: '72000'
        date.timezone=America/Los_Angeles: '0'
        date.time_zone=: America/Los_Angeles
        date.timezone: America/Los_Angeles
        max_execution_time: '60'
        max_input_time: '120'
        memory_limit: 512M
        upload_max_filesize: 512M
        post_max_size: 512M
        display_startup_errors: 'On'
    fpm_ini:
        error_log: /var/log/php-fpm.log
    fpm_pools:
        phpfp_lqqhkodzs395:
            ini:
                prefix: www
                security.limit_extensions: .php
                user: www-user
                group: www-data
                listen: '127.0.0.1:9006'
    composer: '1'
    composer_home: ''
xdebug:
    install: '1'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_connect_back: '1'
        xdebug.remote_enable: '1'
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
blackfire:
    install: '0'
    settings:
        server_id: ''
        server_token: ''
        agent:
            http_proxy: ''
            https_proxy: ''
            log_file: stderr
            log_level: '1'
        php:
            agent_timeout: '0.25'
            log_file: ''
            log_level: '1'
xhprof:
    install: '0'
wpcli:
    install: '0'
    version: v0.21.1
drush:
    install: '0'
    version: 8.0.5
ruby:
    install: '1'
    versions:
        rv_ktys72jhpmga:
            default: '1'
            bundler: '1'
            version: 1.9.3
            gems:
                - deep_merge
                - activesupport
                - vine
python:
    install: '1'
    packages: {  }
    versions: {  }
nodejs:
    install: '0'
    settings:
        version: '5'
    npm_packages: {  }
hhvm:
    install: '0'
    nightly: 0
    composer: '1'
    composer_home: ''
    settings: {  }
    server_ini:
        hhvm.server.host: 127.0.0.1
        hhvm.server.port: '9000'
        hhvm.log.use_log_file: '1'
        hhvm.log.file: /var/log/hhvm/error.log
    php_ini:
        display_errors: 'On'
        error_reporting: '-1'
        date.timezone: UTC
mariadb:
    install: '0'
    settings:
        version: '10.1'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users:
        mariadbnu_xyjha35ucxv7:
            name: dbuser
            password: '123'
    databases:
        mariadbnd_5p13m9s7nxba:
            name: dbname
            sql: ''
    grants:
        mariadbng_xbck4kylseib:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
mysql:
    install: '1'
    settings:
        version: '5.6'
        root_password: '12345'
        override_options: {  }
    adminer: 0
    users:
        mysqlnu_ljbdgnhb5tmf:
            name: ryaa_user
            password: Isaiah
        mysqlnu_xigplljatch0:
            name: nopro_user
            password: et8Ryp79EeS5KRmf
        mysqlnu_8cccxaouedzs:
            name: pma
            password: ''
        mysqlnu_8cccxaouedza:
            name: admin
            password: '12345'
    databases:
        mysqlnd_7dxrldk9z653:
            name: ryaa2
            sql: /home/ryaa/mysql/ryaa2.sql
        mysqlnd_pj9c26loydf7:
            name: qcubed
            sql: /var/www/qcubed/vendor/qcubed/framework/assets/php/examples/mysql_innodb.sql
        mysqlnd_cljntu2xq9cf:
            name: nonpro
            sql: /home/nopro/mysql/nonpro.sql
        mysqlnd_wol7i5w9xfi9:
            name: ryaa_photo
            sql: /home/ryaa/mysql/ryaa_photo.sql
        mysqlnd_r50be6154k5i:
            name: ryaa_data
            sql: /home/ryaa/mysql/ryaa_data.sql
        mysqlnd_o74wldonfkvp:
            name: epaparrc
            sql: ''
    grants:
        mysqlng_dpjpk9mxy30n:
            user: admin@localhost
            table: '*.*'
            privileges:
                - ALL
        mysqlng_5gt58isgjdh5:
            user: pma
            table: '*.*'
            privileges:
                - ALL
        mysqlng_85lnvwxm49ve:
            user: ryaa_user
            table: '*.*'
            privileges:
                - DELETE
                - INSERT
                - SELECT
                - UPDATE
        mysqlng_9xqdzopx1e86:
            user: nonpro_user
            table: 'nonpro.*'
            privileges:
                - DELETE
                - INSERT
                - SELECT
                - UPDATE
postgresql:
    install: '1'
    settings:
        global:
            encoding: UTF8
            version: '9.3'
        server:
            postgres_password: '12345'
    databases:
        postsqlnd_i82x7bq2kluy:
            dbname: qcubed
            owner: qcubed
    users:
        postsqlnu_kv8rv614b6rn:
            username: qcubed
            password: qcubed
            superuser: '1'
    grants:
        postsqlng_p80b6w6p1865:
            role: qcubed
            db: qcubed
            privilege:
                - ALL
    adminer: 0
mongodb:
    install: '0'
    settings:
        bind_ip: 127.0.0.1
        port: '27017'
    globals:
        version: 2.6.0
    databases: {  }
redis:
    install: '0'
    settings:
        port: '6379'
sqlite:
    install: '0'
    adminer: 0
    databases: {  }
mailhog:
    install: '0'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '8025'
        path: /usr/local/bin/mailhog
beanstalkd:
    install: '0'
    settings:
        listenaddress: 0.0.0.0
        listenport: '11300'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: 0
rabbitmq:
    install: '0'
    settings:
        port: '5672'
    users: {  }
    vhosts: {  }
    plugins: {  }
elastic_search:
    install: '0'
    settings:
        version: 2.3.1
        java_install: true
    instances:
        esi_p49prhgxrwhi:
            name: es-01
solr:
    install: '0'
    settings:
        version: 4.10.2
        port: '8984'

**Comment by spekary** [comment]
I think I found a solution to this, and another problem I had, where SimpleXML was not found. PHP is supposed to have XML installed by default, but its not working. The fix on ubuntu was this:

apt-get install php5.6-xml

**Comment by Fynkymynky** [comment]
Glad someone else is having the same issue, I don't think the solution is to install packages manually, it must be something to do with php fpm.

**Comment by spekary** [comment]
I think this has to do with https://github.com/puphpet/puppet-puphpet/pull/39. Or something like it. Just a guess. Looks like people are having problems all over the place with lots of different packages, and it seems to be related to some recent change regarding how packages are specified in PHP. There seems to be a divergence with packages designed for PHP7 vs 5.6 and a need to specifically target the version number when specifying a package.

**Comment by jtreminio** [comment]
Closed by be54496

Fixed in latest release
